src/vs/platform/extensionManagement/test/common/extensionGalleryService.test.ts

```diff
@@ -153,3 +158,51 @@ suite('Extension Gallery Service', () => {
 		return { version, targetPlatform } as IRawGalleryExtensionVersion;
 	}
 });
+
+class MockTelemetryService implements ITelemetryService {
```

Comment:
Please use the NullTelemetryService instead

https://github.com/microsoft/vscode/blob/ade80ca128741e2aa97e836a1ff0ad7cf8d67fa7/src/vs/platform/telemetry/common/telemetryUtils.ts#L46

Reply:
Thanks, fixed it!
What is the best way to send PR, should I talk to you first?
I am still getting familiar with the code base; I don't want to be wasting your time in review :)

Reply:
Either way works. If it is a small one like this, you can go ahead and create a PR like here and ask for review. If its a big change reaching out and discussing will definitely save time for you 👍